spec/features/admin/index_spec.rb

```diff
@@ -8,12 +8,13 @@
       expect(page).to have_content("Admin Dashboard")
     end
     
-    it "should display links to admin/merchants and admin/invoices admin/dashboard" do
+    it "should display links to admin/merchants, admin/invoices, and admin/dashboard" do
       visit admin_path
-      save_and_open_page
+      
       expect(page).to have_link("Dashboard", :href => admin_path)
       expect(page).to have_link("Merchants", :href => admin_merchants_path)
       expect(page).to have_link("Invoices", :href => admin_invoices_path)
     end
   end
-end
\ No newline at end of file
+end
+#save_and_open_page
```

Comment:
Is this just so that you have a "snippet" for later? if so, I'd recommend that you add this snippet to your vs code so that you can call on it anytime with `save` - then we can get rid of the line 20 altogether here:

```
"save_and_open_page": {
		"prefix": "save",
		"body": ["save_and_open_page"],
		"description": "launchy snippet to save and open page"
			}
```

Reply:
Wow, looks good!